### DISCUSSION TITLE
Tips for titanic storms?

### DISCUSSION THREAD

**Discussion by qfmp1012** [niels747/2D-Weather-Sandbox#97] [General]
I know a lot of heat and moisture is needed. but when those conditions are met storms are usually mid, for me at least, What settings can be used along with lots of moisture and heat to create Oklahoma type severe storms.

**Comment by Meso22** [comment]
I use melting heat 1.2 to make stronger cored thunderstorms and evap heat 2.5 for stronger updrafts
